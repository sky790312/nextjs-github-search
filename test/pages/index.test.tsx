import React from 'react'
import { render, withProviders } from '@/test/testUtils'
import Users from '@/pages/'

describe('Home Users page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(withProviders(<Users />), {})
    expect(asFragment()).toMatchSnapshot()
  })
})
