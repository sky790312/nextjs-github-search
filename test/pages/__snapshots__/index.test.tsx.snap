// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Users page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="GlobalStyles__Container-sc-1m5drgn-0 frTxMm"
  >
    <div
      class="GlobalStyles__TextCenterContainer-sc-1m5drgn-1 kwExyG"
    >
      <h3>
        search github repo by keyowrd: 
      </h3>
      <input
        type="text"
      />
    </div>
    <div
      class="GlobalStyles__ListContainer-sc-1m5drgn-2 lDTWr"
    />
    <div
      class="GlobalStyles__TextCenterContainer-sc-1m5drgn-1 LoadingSection__LoadingSectionWrapper-ekcqrm-0 kwExyG iSBBve"
    >
      <button
        class="LoadingSection__Button-ekcqrm-1 iPqPCv"
      >
        Load More
      </button>
    </div>
  </div>
</DocumentFragment>
`;
